### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by kikimora52** [nilsteampassnet/TeamPass#3920] [Q&A]
I would like to start by stating I believe this is one of the better products on the market for team-based password management. At my previous company we developed our own tool. I wish I had know of this project at the time.

That said I am having issues getting this running. I have tried 3.0.10, 3.0.9 and 3.0.8. Each presents their own issues. I am using a docker install with the nginx proxy with acme-companion as my deployment method. LDAP seemed to work great and so I am using this for user management. 

In 3.0.10 when SSL is introduced I see a banner on the screen informing of inconsistent config I did not wish to proceed beyond. 

In 3.0.9 when clicking on Tasks I receive an Ajax popup complaining of a missing table or column in the DB.

In 3.0.8 I was able to create one user imported from LDAP. The OTP printed on screen I handed it to the user and all was well. However, I could not create any further user accounts. The account seemed construct fine, however, the OTP came back as bad code.

All versions seem to a fatal php issue while evaluating a cron time table. No accounts get created until I comment out this check.

Is there a version I can standup today and get working? As I mentioned, this is the perfect tool for this team. I want this to work.

Cheers

**Comment by nilsteampassnet** [comment]
@kikimora52 
Thank you for your constructive message.
I would say that the most recent commit is the best one to use, so you should go on with 3.0.10 that I will soon release (this week).
Can you please give me more details regarding this SSL issue you get?
What options regarding SSL are you enabling?
thank you

**Comment by Arash9703** [comment]
I would also like to suggest a small update, if possible. It would be beneficial to have a feature that allows the addition of multiple or even all LDAP users to Teampass, rather than adding them one by one. I understand that the key encryption policy might make this seem impossible. It would be a time-saving plus point and would be appreciated by admins.
